firmware/user/modes/mode_tunernome.c

```diff
@@ -534,7 +614,7 @@ void ICACHE_FLASH_ATTR tunernomeButtonCallback( uint8_t state __attribute__((unu
                             grn = 0;
                             blu = 255;
                         }
-                        loc = (NUM_STRINGS + 1 - (tunernome->currentMode % 6)) % 6;
+                        loc = (NUM_STRINGS + 1 - (tunernome->curTunerMode % 6)) % 6;
```

Comment:
user/modes/mode_tunernome.c: In function 'tunernomeUpdate':
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_0' not handled in switch [-Wswitch-enum]
             switch(tunernome->curTunerMode)
             ^
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_1' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_2' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_3' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_4' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_5' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_6' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_7' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_8' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_9' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_10' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'SEMITONE_11' not handled in switch [-Wswitch-enum]
user/modes/mode_tunernome.c:396:13: warning: enumeration value 'MAX_GUITAR_MODES' not handled in switch [-Wswitch-enum]

Reply:
I had forgotten that was still in there when I pushed. Already fixed it on my branch!